scipy/stats/_distn_infrastructure.py

```diff
@@ -1660,9 +1660,17 @@ class rv_continuous(rv_generic):
     a : float, optional
         Lower bound of the support of the distribution, default is minus
         infinity.
+        Note: This value is unaffected by the `loc` and `scale` parameters.
+        It represents the theoretical lower bound of the unscaled distribution.
+        To calculate the adjusted lower bound after applying `loc` and `scale`,
+        use the `support()` method.
     b : float, optional
         Upper bound of the support of the distribution, default is plus
         infinity.
+        Note: This value is unaffected by the `loc` and `scale` parameters.
+        It represents the theoretical lower bound of the unscaled distribution.
+        To calculate the adjusted lower bound after applying `loc` and `scale`,
+        use the `support()` method.
```

Comment:
This is the "Parameters" section.
<img width="858" alt="image" src="https://github.com/user-attachments/assets/243e4e0c-2738-476f-9a5f-73007714145d" />

But I think what needs to change is the "Attributes" section. How about adding there:
```
    a, b : float, optional
        Lower/upper bound of the support of the unshifted/unscaled distribution.
        This value is unaffected by the `loc` and `scale` parameters.
        To calculate the support of the shifted/scaled distribution,
        use the `support` method.
```